packages/sdk/package.json

```diff
@@ -51,16 +51,18 @@
   "dependencies": {
     "@azure/identity": "^2.0.1",
     "@azure/msal-node": "~1.1.0",
+    "@microsoft/microsoft-graph-client": "^3.0.1",
     "axios": "^0.24.0",
-    "botbuilder": ">=4.15.0-rc0 <5.0.0",
-    "botbuilder-dialogs": ">=4.15.0-rc0 <5.0.0",
+    "botbuilder": ">=4.15.0 <5.0.0",
```

Comment:
I remember @dilin-MS 's previous investigation suggests putting botbuilder to peerDependencies to avoid the bot sdk version conflict issue (user references a version that is different than the version referenced by our SDK). Can we move the bot sdks to peerDependencies?

Reply:
I moved 'tedious' to peerDependencies in previous PR and met a problem when building Azure Function. (Please check [this bug](https://msazure.visualstudio.com/9660fff2-2363-48b0-9e15-64df2283e932/_workitems/edit/12664346)).
The reason is that type definition file referenced interfaces/classes in tedious. Thus I move it back to devDependencies because npm6 doesn't install peerDependencies by default.
Bot framework library has the same issue and I remember @dilin-MS chose to use this solution.